src/main/java/org/broadinstitute/hellbender/tools/variantdb/nextgen/ExtractCohortFilterRecord.java

```diff
@@ -48,4 +49,7 @@ public ExtractCohortFilterRecord(GenericRecord genericRecord) {
 
     public String getAltAllele() { return this.altAllele; }
 
+    public String toString() {
+        return ReflectionToStringBuilder.toString(this);
```

Comment:
🤩

Reply:
Haha..I was sick of the `ExtractCohortFilterRecord@12318f` coming out in the logger messages :D